cmd/editor_utils.go

```diff
@@ -0,0 +1,45 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+// runEditor opens the specified file in the user's preferred editor.
+// It gets the editor from the EDITOR environment variable.
+// If EDITOR is not set, it falls back to "nano".
+// If EDITOR is set to an empty string, it fails with an error.
+// Returns an error if the editor cannot be found or executed.
+func runEditor(filePath string) error {
+	editor, exists := os.LookupEnv("EDITOR")
+
+	if exists && editor == "" {
+		return fmt.Errorf("EDITOR environment variable is set to empty string")
+	}
+
+	if !exists {
+		if *debug {
+			fmt.Fprintln(os.Stderr, "No editor set. Set $EDITOR to your preferred editor")
+		}
+		editor = "nano"
+	}
```

Comment:
I think that failing with an error might be better than assume that nano is installed or not be in PATH directly
would fail if EDITOR is empty or unset